.github/workflows/update-tf-versions.yml

```diff
@@ -30,4 +30,29 @@ jobs:
           commit-message: "doc: Updates Terraform Compatibility Matrix documentation"
           delete-branch: true
           branch: terraform-compatibility-matrix-update
-          body: "Automatic updates for Terraform Compatibility Matrix documentation. **Action Required**: Update test-suite.yml, .tools-version files, and TF_VERSION_LATEST GitHub environment variable if needed."
+          body: "Automatic updates for Terraform Compatibility Matrix documentation. **Action Required**: Update TF_VERSION_LATEST GitHub environment variable if needed."
```

Comment:
if this PR is being generated with this PR:  https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2756/files

I think it's better to keep:  schedule_terraform_matrix: '["${{ vars.TF_VERSION_LATEST }}"]'
so not changing it to:   schedule_terraform_matrix: '["1.9.x"]'

Reply:
also not sure about the change to .tool-version from: terraform 1.9.7
to: terraform 1.9.0

not sure why it's being downgraded, in fact it would be great if it upgrades to 1.9.8 that is the latest version

Reply:
can we also update these 2 files? 
https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/scripts/generate-doc.sh#L35
https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/scripts/generate-docs-all.sh#L29

Reply:
>I think it's better to keep: schedule_terraform_matrix: '["${{ vars.TF_VERSION_LATEST }}"]'
so not changing it to: schedule_terraform_matrix: '["1.9.x"]'

Could you share more on why it shouldn't be updated? Currently TF_VERSION_LATEST is set to 1.9.x in the repo so it won't really make a difference so I think it's better to automate it to reduce one manual dependency.


>also not sure about the change to .tool-version from: terraform 1.9.7
to: terraform 1.9.0

It looks like a downgrade right now because we have been manually maintaining that. The idea is that these updates should happen automatically when a new major TF version is released. We're currently not keeping track of minor versions.

Reply:
added a comment to the PR with context

Reply:
@lantoli updated to use the latest patch version wherever applicable. This is the new PR: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2759/files 

Yet to update the occurrences of TF_VERSION_LATEST in acceptance-tests and examples.yml, I'll create a ticket for that if I'm not able to get that to work by EOD.

Reply:
nice!

Reply:
Created https://jira.mongodb.org/browse/CLOUDP-281749 to replace TF_VERSION_LATEST